api/core/v2/adhoc.proto

```diff
@@ -14,6 +14,9 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.testgen_all) = true;
 
 message AdhocRequest {
+  option (gogoproto.face) = true;
+  option (gogoproto.goproto_getters) = false;
+
```

Comment:
Do you have a 2 sentences rationale behind the need set these options?
I haven't read the gogoproto manual so it's not obvious what that is to me ;)

Reply:
Long story short, I these attributes missing were causing a bug but turns out it didn't. I reverted these changes to prevent any weird bug!